app/components/provider_interface/reference_with_feedback_component.rb

```diff
@@ -41,7 +41,7 @@ def name_row
     def email_address_row
       {
         key: 'Email address',
-        value: mail_to(email_address),
+        value: mail_to(email_address, email_address, class: 'govuk-link'),
```

Comment:
@duncanjbrown Am assuming this is correct. I copied the code used for email in personal details summary.

Reply:
Looks good. We should really have a `govuk_mail_to` I reckon

Reply:
It’s actually possible to [style links like this by default](https://frontend.design-system.service.gov.uk/sass-api-reference/#govuk-global-styles), using `$govuk-global-styles: true;` in `application.scss`. That said, it’s probably best to use one approach or the other, not a mix of the two.